paasta_tools/utils.py

```diff
@@ -2367,6 +2369,19 @@ def get_log_reader(self) -> LogReaderConfig:
                 % self.directory
             )
 
+    def get_log_readers(self) -> List[LogReaderConfig]:
```

Comment:
just verifying: this will eventually be deleted in favor of having `get_log_reader()` return the vector reader?

Reply:
if we migrate all logs to a single pipeline, then yes. Until then we would use `get_log_readers()` i think